hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnUtils.java

```diff
@@ -724,4 +726,22 @@ public static void doIfNotInterrupted(@NonNull final InterruptibleRunnable runna
     public static KeyList getCompositeList(final Key key) {
         return key.hasKeyList() ? key.getKeyList() : key.getThresholdKey().getKeys();
     }
+
+    /**
+     * Returns the contents of the resource at the given location as a string.
+     *
+     * @param loc the location of the resource
+     * @return the contents of the resource as a string
+     */
+    public static String resourceAsString(@NonNull final String loc) {
+        try {
+            try (final var in = TxnUtils.class.getClassLoader().getResourceAsStream(loc);
+                    final var bridge = new InputStreamReader(requireNonNull(in));
```

Comment:
is this the way spotless left the indentation?  odd.

Reply:
Spotless has a mind of its own. 🤷